pkg/updateengine/client.go

```diff
@@ -80,7 +80,16 @@ func (c *Client) Close() error {
 	return nil
 }
 
+// ReceiveStatuses sends status updates from the update_engine along a provided channel.
```

Comment:
nit: Can we re-arrange to the primary focus is still on channel receiving?

"ReceiveStatuses receives signal messages from dbus and sends them as Statues on the rcvr channel, until the stop channel is closed. An attempt is made to get the initial status and send it on the rcvr channel before receiving starts."